react/Header/UserAccountMenu/UserAccountMenu.less

```diff
@@ -76,10 +76,10 @@
   }
 }
 
-.icon.jobSearch { color: @sk-highlight; }
+.icon.jobSearch { color: @sk-blue-lighter; }
```

Comment:
This actually changes the colour—should this be a release?

Reply:
I was happy not to trigger it, but if you want it to roll out we can. Thoughts?

Reply:
I was going to suggest releasing it, but I'm probably not so keen on the release noise. We can leave it.